pkg/virt-controller/services/template.go

```diff
@@ -858,7 +858,7 @@ func (t *templateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, t
 		if vcpus != 0 && cpuAllocationRatio > 0 {
 			val := float64(vcpus) / float64(cpuAllocationRatio)
 			vcpusStr := fmt.Sprintf("%g", val)
-			if val < 0 {
```

Comment:
I am just curious here. How can vcpus be 0? (line 858)

Reply:
When it's not provided? Not sure, but it's better to check.